data/102/063/209/102063209.geojson

```diff
@@ -10,6 +10,12 @@
     "geom:latitude":47.733449,
     "geom:longitude":9.400789,
     "iso:country":"DE",
+    "label:deu_x_preferred_longname":[
+        "Landkreis Bodenseekreis"
```

Comment:
Is this redundant, like just say `Bodenseekreis`?

Reply:
Hmm, yes I think the double "kreis" is redundant here.. I'll crawl these changes to see if "kreis" exists in th e`wof:name` and, if so, QA and drop the placetype if appropriate..